CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+# 19.0.0 [#589](https://github.com/openfisca/openfisca-core/pull/585)
+
+#### Breaking changes
+
+* Technical change
```

Comment:
"Technical change" is one type of change for country packages. 
I'm not sure it says much in the context of core. I'd remove this line.